libs/common/src/config/config.service.ts

```diff
@@ -4,28 +4,20 @@ https://docs.nestjs.com/providers#services
 
 import { Injectable } from '@nestjs/common';
 import { ConfigService as NestConfigService } from '@nestjs/config';
-import { EnvironmentDto } from '..';
+import { ChainEnvironment } from '..';
 import { ICapacityLimit } from '../interfaces/capacity-limit.interface';
 
 export interface ConfigEnvironmentVariables {
-  ENVIRONMENT: EnvironmentDto;
+  CHAIN_ENVIRONMENT: ChainEnvironment;
   IPFS_ENDPOINT: URL;
   IPFS_GATEWAY_URL: URL;
   IPFS_BASIC_AUTH_USER: string;
   IPFS_BASIC_AUTH_SECRET: string;
   REDIS_URL: URL;
   FREQUENCY_URL: URL;
   PROVIDER_ID: string;
-  BLOCKCHAIN_SCAN_INTERVAL_MINUTES: number;
-  QUEUE_HIGH_WATER: number;
-  WEBHOOK_FAILURE_THRESHOLD: number;
-  HEALTH_CHECK_SUCCESS_THRESHOLD: number;
-  WEBHOOK_RETRY_INTERVAL_SECONDS: number;
-  HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS: number;
-  HEALTH_CHECK_MAX_RETRIES: number;
   PROVIDER_ACCOUNT_SEED_PHRASE: string;
   CAPACITY_LIMIT: ICapacityLimit;
-  FILE_UPLOAD_MAX_SIZE_IN_BYTES: number;
```

Comment:
Even though this is not being used directly from config I think it would help to keep it around for future and consistency between the env variables and config fields . This configuration has being used directly inside `ApiController` since I couldn't find an easy way to inject the config service inside there.

Reply:
fixed--I restored this to the environment, and found a way to honor it from the ConfigService using async default module options

Reply:
Awesome, I would lower this value for test and check to ensure it does limit the uploaded file and works as expected

Reply:
Yes, I did that--errored as expected.